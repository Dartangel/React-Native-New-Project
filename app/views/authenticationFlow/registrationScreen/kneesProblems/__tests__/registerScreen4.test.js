import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import KneesProblems from '../kneesProblems';

describe('KneesProblems', () => {
    test('KneesProblems snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<KneesProblems />);
        expect(snapshot).toMatchSnapshot();
    });
});