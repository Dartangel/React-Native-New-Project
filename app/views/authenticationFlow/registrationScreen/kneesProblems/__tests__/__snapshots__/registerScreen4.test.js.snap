// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KneesProblems KneesProblems snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/src/img/splashScreen1.jpeg",
    }
  }
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Header
    header="Do you have a problems with knees?"
    subHeader="Choose a workout according to your condition"
  />
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <ChoosePlan
      title="I am OK"
    />
    <ChoosePlan
      title="Without jumping"
    />
    <ChoosePlan
      title="Low impact"
    />
    <View
      style={
        Object {
          "alignItems": "center",
        }
      }
    >
      <NextButton
        onPress={[Function]}
        title="Next"
      />
    </View>
  </View>
</ImageBackground>
`;
